app-i18n/transifex-client/files/transifex-client-timestamp.patch

```diff
@@ -0,0 +1,14 @@
+diff -Naur ../transifex-client-0.14.2.orig/tests/test_project.py ./tests/test_project.py
```

Comment:
I've tidied up the patch a bit - see https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions and ab3ec2595d0398dae2a2856122229b4454fe6a54. Thank you!